content/shmem_atomic_add.tex

```diff
@@ -53,14 +55,17 @@
 }
 
 \apinotes{
-    The term remotely accessible is defined in Section \ref{subsec:memory_model}.
+    As of OpenSHMEM~1.4, the use of \FUNC{shmem\_add} has been deprecated
+    and should be expected to be removed in a future specification.
+    Its behavior and call signature is identical to the replacement
+    interface, \FUNC{shmem\_atomic\_add}.
```

Comment:
The deprecation statement is present for ```shmem_add``` and a couple others, but not all of deprecated atomic routines.  This seems inconsistent?  It's also a little weird that ```shmem_add``` is mentioned here, but the function itself (i.e. its prototype) is absent from the specification.